cmd/algocfg/profileCommand.go

```diff
@@ -66,8 +66,8 @@ var (
 		},
 	}
 
-	relay = configUpdater{
-		description: "Relay consensus messages across the network and support catchup.",
+	wsRelay = configUpdater{
```

Comment:
I guess you renamed this to be explicit that it's for ws network, but it's a change of behavior. I might keep this as `relay` and update the description, but I can see how it's clearer the purpose here.

Reply:
Given its a use it once type thing, I thought this okay. If we are feeling strongly, can have both `relay` and `wsRelay` have the same behavior.